.github/workflows/manual-release.yml

```diff
@@ -0,0 +1,25 @@
+name: Attach artifacts to release (manually created)
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Prepare Release
+        run: |
+          curl --insecure --silent --location --write-out '%{http_code}' --output ./piper_master https://github.com/SAP/jenkins-library/releases/latest/download/piper_master
+          cp ./piper_master ./piper
+      - uses: SAP/project-piper-action@master
+        with:
+          piper-version: latest
+          command: githubPublishRelease
+          flags: --token ${{ secrets.GITHUB_TOKEN }} --version latest --assetPath ./piper_master
```

Comment:
Isn't this adding the `piper_master` to the same release where it was downloaded from?

Reply:
No, this action triggers when a release was manually created, so that is a new one.

Potential issue/question: Does the download link work in this scenario as now (for a short period of time) the "latest" release has no attached binaries (which is what we're trying to add here).

Another question: How can we get the same generated release notes as we get for weekly releases?

Reply:
CF releases/workflow runs in my testing repo https://github.com/fwilhe/glowing-octo-couscous/releases